config/config.exs

```diff
@@ -38,6 +38,12 @@ config :logger, :console,
   ],
   colors: [enabled: true]
 
+# Faucet rate limit in Number of transactions
+config :archethic, :faucet_rate_limit, 3
```

Comment:
Will it mean in this configuration no more than 3 tx/hour ?

Reply:
Yeah! that means we are limiting 300 UCO per hour. Just To Test It I wrote that to `3`